src/latent-processes.jl

```diff
@@ -1,27 +1,43 @@
 """
-    random_walk(n, ϵ_t = missing; latent_process_priors = (σ_RW_dist = truncated(Normal(0., 0.05), 0., Inf),), ::Type{T} = Float64) where {T <: Real}
+    random_walk(n, ϵ_t = missing; latent_process_priors = (var_RW_dist = truncated(Normal(0., 0.05), 0., Inf),), ::Type{T} = Float64) where {T <: Real}
```

Comment:
Why do we want to define this in terms of the variance and not the standard deviation?

Reply:
Its standard for HalfNormals to be priors for variance rather than std. Since it doesn't make much different I think going with the literature norm is probably best upon reflection.

Reply:
okay that makes sense though I would note its not standard in other PPLs (i.e. stan) and so if aiming for interoperability this could be confusing. I am happy with either choice though.

Reply:
Ah right, I thought that was the stan default.